src/foundry_dev_tools/utils/token_provider.py

```diff
@@ -1,137 +0,0 @@
-"""Token Provider for common UPTIMIZE AppService."""
```

Comment:
We have to leave a code path in (for compatibility reasons), where we check if os.environ['APP_SERVICE_TS'] is present and if yes we automatically try both token providers in this file.